viz-lib/src/visualizations/chart/Editor/ChartTypeSelect.jsx

```diff
@@ -39,12 +40,10 @@ export default function ChartTypeSelect({ hiddenChartTypes, ...props }) {
   );
 }
 
-ChartTypeSelect.defaultProps = {
-  hiddenChartTypes: [],
+ChartTypeSelect.propTypes = {
+  hiddenChartTypes: PropTypes.arrayOf(PropTypes.oneOf(result.map(({ type }) => type))),
```

Comment:
```suggestion
  hiddenChartTypes: PropTypes.arrayOf(PropTypes.oneOf(map(result, "type"))),
```

Please rename it as @kravets-levko already mentioned, and just in case you want to use Lodash map instead 🙂 (it is already imported anyway)

Reply:
Wow, I didn't know that's supported by `_.map` 😅  I used this syntax with `_.filter`, `_.sortBy` but missed that other functions may support it 🙂